### ISSUE TITLE
FOBs missing options

### ISSUE THREAD

**Issue by Doretz** [KillahPotatoes/KP-Liberation#633]
### Basic Information
FOB has broken actions
**Mission version:**
v0.96.4
**Map used:**
Altis
**Mods used:**
CBA_A3
RHSAFRF
RHSUSAF
ace
RHSGREF
JSRS SOUNDMOD
CUP Weapons
NIArms All in One
RHSSAF
ACE 3 Extension (Animations and Actions)
Enhanced Movement
Advanced Sling Loading
CUP Terrains - Core
CUP Terrains - Maps
CUP Terrains - CWA
Advanced Towing
F/A-181E Black Wasp II
FIR AWS(AirWeaponSystem)
A-10 Warthog
Barrett M107
Multi-play Uniforms
Drongo's Active Protection System
WMO - Walkable Movable Objects
Project OPFOR
ACE Compat - RHS USAF
ACE Compat - RHS AFRF
ACE Compat - RHS GREF
Complementary Police Weapons
BloodLust
MCC Sandbox 4
Dagger Weapons, Gear, Vehicles, & Factions mod for ACE3
NIArms All In One - CUP Compatibility
NIArms All In One - RHS Compatibility
NIArms All In One - ACE Compatability
Complementary Special Pistols 
MLO Uniforms
**Dedicated or local game:**
Local
**Source of the mission file (URL):**

### Individual Things
**Did you edit anything within the mission files?**
No

**If yes, what did you edit exactly?**

### Short Issue Description
I know this issue has been talked about but this won't fix itself, the menus are still broken after restarting games and resetting everything, what am I supposed to do? We were playing on a LAN server with my friend, when we restarted it, both FOBs we had would not allow us to do anything, couldn't go into arsenal or anything, none of the scroll menu selections were showing up like redeploy or arsenal. We also had an error earlier that game with the FOB hunt side mission, which may have caused it. Can't remember exactly what it was as it was hours before but it made my storage view desync from his and mine would not change to the new updated values for things, such as storage, fuel, ammo, or enemy awareness. The weird thing is that the vehicles around the FOBs had "Recycle" and "Unflip" from the mission.

### Expected Behaviour

The FOBs should have arsenal, redeploy, halo jump, and all the other things

### Steps to Reproduce the Issue

Don't really know how to reproduce it as it seemed random. 

### Screenshots, Serverlog or any other helpful Information
![image](https://user-images.githubusercontent.com/25855408/61104491-8eadd880-a444-11e9-9495-5794cdae27ed.png)
![image](https://user-images.githubusercontent.com/25855408/61104508-9f5e4e80-a444-11e9-9b99-6ae1d4aea398.png)
![image](https://user-images.githubusercontent.com/25855408/61104514-a7b68980-a444-11e9-8ca7-df1bcd5bb48b.png)

**Comment by Doretz** [comment]
Just in case you read the basic information and close this, I looked 2 years in the past and couldn't find any information on this, the newest issue opened about this was closed with "wontfix" but is my save ruined? Because it won't fix itself.

**Comment by Wyqer** [comment]
This is in 99% caused by a corrupted intel value.
You'll have something like this in your rpt (at least it'll mention `resources_intel`)
```
Error in expression <at_readiness
|| _resources_intel_old != resources_intel
|| _infantry_cap_old != >
Error position: <resources_intel
|| _infantry_cap_old != >
Error Undefined variable in expression: resources_intel
```

Set your intel value to a valid number via the Debug console -> `resources_intel = 20;` (or any other number), execute in on server and you're fine again.

This is already fixed for 0.96.5 (which will be released along with the Contacts DLC). If you want to apply the fix on your own, look at this commit:
https://github.com/KillahPotatoes/KP-Liberation/commit/35995074fadea24698937dfaa8d3dacc9b5e334d
No, I won't provide help to do this edit in your file on your server/machine, it's easy enough.

To avoid this to happen (without the fix) don't close the secondary mission dialog on your own (it will close automatically), after you've clicked the button to start a mission.
